lib/action_logic/action_validation.rb

```diff
@@ -6,27 +6,31 @@ module ActionLogic
   module ActionValidation
     module ClassMethods
       def validates_before(args)
-        @validates_before = args
+        @validates_before = args.merge(raise_action_logic_exception: true)
```

Comment:
Will eventually eliminate this dupe, but for now I just wanted to see what it looked like to use the `validations` hash to signal to the validation classes whether to raise an exception or not.